src/ray/protobuf/common.proto

```diff
@@ -196,6 +196,9 @@ message TaskSpec {
   bool placement_group_capture_child_tasks = 20;
   // Environment variables to override for this task
   map<string, string> override_environment_variables = 21;
+  // Whether or not to skip the execution of this task and only use this task for
+  // task ordering purpose.
```

Comment:
Can you include more context in this comment? It's hard to understand what "task ordering purpose" means.